src/scripts/underground/Mine.ts

```diff
@@ -315,6 +315,8 @@ class Mine {
         this.skipsRemaining(mine.skipsRemaining ?? this.maxSkips);
 
         Underground.showMine();
+        // Check if completed in case the mine was saved after completion and before creating a new mine
+        setTimeout(Mine.checkCompleted);
```

Comment:
Although it isn't required, can you please provide the delay so it is more clear to anyone looking at the code? Can you also please add a comment that the `setTimeout` is required to push it back one event cycle so that `App.game` is available? It won't be required when we're on TypeScript modules, and would be good to clean it up
```suggestion
        // TODO: Remove setTimeout after TypeScript module migration is complete. Needed to that `App.game` is available
        setTimeout(Mine.checkCompleted, 0);
```